src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -569,6 +570,16 @@ public void selectQuestionNumber(int qnNumber, int newQnNumber) {
         selectDropdownByVisibleValue(qnNumSelect, String.valueOf(newQnNumber));
     }
 
+    public boolean isSelectQuestionNumberEnabled(int qnNumber) {
+        WebElement qnNumSelect = browser.driver.findElement(By.id("questionnum-" + qnNumber));
```

Comment:
That is what I meant. I believe all three methods could use that. 
```
Select qnNumSelect = new Select(getSelectQuestionNumberDropdown(qnNumber));
``` 
can be done instead.